src/index.html

```diff
@@ -1,17 +1,171 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="http://localhost:8080/">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              minlength="3"
+              maxlength="30"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              minlength="2"
+              maxlength="20"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="how__old"
+              min="1"
+              max="100"
+              value="12"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Full date of birth:
+            <input type="date" name="date" >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="I accept" >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form__field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              autocomplete="off"
+              multiple
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Password:
+            <input type="password" name="password" >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form__field">
+          <label>
+            Do you love cats?
+            <input type="radio" 
+            name="cats" 
+            value="y" 
+            >
+            Yes
+          </label>
+          <label>
+            <input type="radio" 
+            name="cats" 
+            value="n" 
+            >
+            No
```

Comment:
```suggestion
            <input
              type="radio" 
              name="cats" 
              value="n" 
            >
             No
```
fix all the inputs according to checklist style